src/transforms/dropExtrema.jl

```diff
@@ -0,0 +1,27 @@
+# Licensed under the MIT License. See LICENSE in the project root.
+# ------------------------------------------------------------------
+
+"""
+        DropExtrema(low, high)
+    
+    Applies the DropExtrema transform to remove rows with values outside the interval [low, high].
+    The DropExtrema transform drops extreme values in both lower and upper tails.
+    
+        DropExtrema(col, low, high)
+    
+    Applies the DropExtrema transform to the specified column `col` with interval [low, high].
+    
+    # Examples
+    ```julia
+    DropExtrema(0.01, 0.99)
+    DropExtrema(:column_name, 1.0, 10.0)
+    
+"""
+function drop_extrema(df::DataFrame, column_name::Symbol, low::Float64, high::Float64)
```

Comment:
You need to implement the same functions that are implemented for other transforms. See the developer guide for more information: https://juliaml.github.io/TableTransforms.jl/stable/devguide/

Reply:
yeah sir I will check it